README.md

```diff
@@ -101,7 +117,7 @@ To specify the parameters, we put them a [yaml](http://www.yaml.org/start.html)
 configuration file, in this case called `pnlpipe_params/std.params`. To make a
 default version of this file, run
 
-    ./pnlpipe std init
+    ./pnlpipe std init # ignore any warnings related to dtype or size
```

Comment:
Are you still getting these errors? I thought they were supposed to be fixed in the latest numpy.

Reply:
Yeah, I think that's why I put that additional comment.